core/object/message_queue.cpp

```diff
@@ -28,6 +28,10 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
+
+#include <stdio.h>
+
+
 #include "message_queue.h"
 
 #include "core/config/project_settings.h"
```

Comment:
```suggestion
#include "message_queue.h"

#include "core/config/project_settings.h"

#include <stdio.h>
```
Keep the order and structure